CMakeLists.txt

```diff
@@ -516,8 +516,7 @@ if (HSA_USE_AMDGPU_BACKEND AND HSA_LLVM_BIN_DIR)
 
   # create a symlink to the LLVM binaries for the amdgpu backend in the project directory
   add_custom_target(lightning_bin
-    COMMAND mkdir -p "${PROJECT_BINARY_DIR}/llvm/bin"
-    COMMAND cp -rp "${HSA_LLVM_BIN_DIR}/*" "${PROJECT_BINARY_DIR}/llvm/bin/"
+    COMMAND ln -sf "${HSA_LLVM_BIN_DIR}" "${PROJECT_BINARY_DIR}/llvm/bin"
```

Comment:
copying binaries does not work, and breaks build if llvm is in non-default location.
$ chrpath -l bin/llvm-link
bin/llvm-link: RUNPATH=$ORIGIN/../lib

resulting in "failed to load library" errors during build
the script either needs to change rpath, or copy libraries to a corresponding location as well. or fix the packaging script, no idea which one is the easiest.
I'd argue that packaging binaries without corresponding libraries is a bad idea anyway